planemo/git.py

```diff
@@ -98,6 +98,9 @@ def command_clone(
         cmd.append("--mirror")
     if branch is not None:
         cmd.extend(["--branch", branch])
+        cmd.extend(["--depth", "1"])
+        if not src.startswith("file://"):
+            src = f"file://{src}"
```

Comment:
Why is this needed?
In case, I think this should be done by the caller of this function.

Reply:
https://stackoverflow.com/questions/40383230/git-clone-warning-depth-is-ignored-in-local-clones-use-file-instead

Reply:
Ah, thanks! In that case it could be conditional on `depth is not None` and not being a URL, but maybe not worth the effort.